### DISCUSSION TITLE
Updating LFRic kgo

### DISCUSSION THREAD

**Discussion by Adrian-Lock** [MetOffice/simulation-systems#233] [LFRic]
I'm afraid I'm struggling to work out the right syntax to do this.  The working practices on https://metoffice.github.io/simulation-systems/WorkingPractices/kgo.html give the command as:
python3 ./rose-stem/bin/update_branch_kgos.py -s <suite name> -w <path to working copy>
I've got my working copy, vn1.1_kgo_for_bm_cflmax, in a directory (so I think <path to working copy> is easy enough) but I can't work out what is meant by <suite name>?  I want it to update all the kgos that I've changed.  I'd find an example very useful, if that were possible.

**Comment by james-bruten-mo** [comment]
Hi Adrian, the command is `python3 ./rose-stem/bin/update_branch_kgos.py -s ROSE-STEM-SUITE-NAME -w PATH-TO-WC`, so if you had run rose stem in that working copy you mention, then running from the top level of the working copy might look something like `python3 ./rose-stem/bin/update_branch_kgos.py -s vn1.1_kgo_for_bm_cflmax/run1 -w .`

**Reply by Adrian-Lock** [reply]
Great, thanks, that worked.  It was the /run1 I was missing.
I'll make a note of it for myself but I'd suggest that would be a really useful addition to the guidance page.
